python/ray/services.py

```diff
@@ -1163,6 +1163,18 @@ def start_raylet(redis_address,
     else:
         java_worker_command = ""
 
+    try:
+        import pydevd
+        is_pydevd = True
+        pydevd_setup = pydevd.SetupHolder.setup
+    except ImportError:
+        is_pydevd = False
+
+    if is_pydevd:
```

Comment:
instead of declaring and checking is_pydevd, why don't you put all code inside of the try?